### PR TITLE
Dockerization of the project

### PR DESCRIPTION
**Pull request by Areedtz**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+version: '3'
+services:
+  codebase:
+    # Set CURRENT_ID by running CURRENT_UID=$(id -u):$(id -g) docker-compose up
+    user: ${CURRENT_UID}
+    build: .
+    volumes:
+    - ./src:/code
+    command: tail -f /dev/null
+
```
Comment (QuincySharp) [inline docker-compose.yml]: new line?

File: requirements.txt [hunk 2]
```diff
@@ -3,9 +3,9 @@ pytest==4.2.1
...
-h5py
-opencv-python
-tensorflow
-numpy
+scipy==1.2.1
+h5py==2.9.0
+opencv-python==4.0.0.21
+tensorflow==1.13.1
+numpy==1.16.2
 keras
```
Comment (EmilHenriksen) [inline requirements.txt]: Not for keras?

File: docker-compose.yml (hunk 1 above)
Comment (LukasBHolst) [reply docker-compose.yml]: yes

Comment (QuincySharp) [reply docker-compose.yml]: okay

Comment (Areedtz) [reply docker-compose.yml]: Shouldn't matter :)

File: requirements.txt (hunk 2 above)
Comment (Areedtz) [reply requirements.txt]: Couldn't find a version for it. You're very welcome to add a version if you know what version we're using.

**Pull request merged**